docs/flatcar-linux/kvm-libvirt.md

```diff
@@ -186,6 +186,8 @@ ssh-add ~/.ssh/id_rsa
 ssh-add -L
 ```
 
+To login by SSH, use the default user `core` -> [ssh.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/master/kvm-libvirt/flatcar-linux/kubernetes/ssh.tf#L8)
```

Comment:
```suggestion
If you want to login by SSH later for debugging purposes, use the default user `core`.
```

Reply:
Shall I commit your suggestion and go on with merging?

Reply:
Thanks @pothos . I committed your suggestion.